pathd/path_pcep_pcc.c

```diff
@@ -651,6 +667,9 @@ void pcep_pcc_pathd_event_handler(struct ctrl_state *ctrl_state,
 		PCEP_DEBUG("%s Candidate path %s removed", pcc_state->tag,
 			   path->name);
 		path->was_removed = true;
+		/* Removed as response to a PcInitiated 'R'emove*/
+		/* RFC 8291 #5.4 LSP Deletion*/
+		path->do_remove = path->was_removed;
```

Comment:
Wrong RFC number. Please adjust the comment.

However, I don't understand how you get the `R` flag form the PcInitiated message. Can you point me in the code where this is done ?

Reply:
1.- In pceplib it's parsed in pcep_decode_obj_srp .
2.- In pathd control thread converted to path struct in pcep_lib_parse_srp and check for add or delete and for errors in handle_pcep_lsp_initiate .
3.- Finally in pathd main thread to update config, frr,etc in pcep_main_event_initiate_candidate
(the support in pceplib was previously done)

Reply:
comment fixed.